JBPM-9840 unable to stop KIE container via BC UI with error Not suppo… (#3855)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.59.0.Final/container-stop.adoc

[id='smart-router-tls-759']

> = Container stop

Reviewer: ```suggestion
= Stop a container 
```

> When user want to stop a container it has active process instances, the user can get the same result as the following figure.

Reviewer: ```suggestion
You can now stop a container in {PRODUCT}. When you want to stop a container, the container contains active process instances, and you can get the results as shown in the following figure.

image::ReleaseNotes/stop_container.png[align="left", title="Stop container"]

```

> You can now stop a container in business-central, When you want to stop a container, the container contains active process instances, and you can get the results as shown in the following figure.

Reviewer: ```suggestion
You can now stop a container in business-central, When you want to stop a container, the container contains active process instances, and you can get the results as shown in the following figure.

image::ReleaseNotes/stop_container.png[align="left", title="Stop container"]
```

Author: done

> If user want to start the container, user will see same result as the Figure 2. After user re-deploy the container, user will get accurate results.

Reviewer 2: ```suggestion
If user want to start the stopped container, user will see same result as the Figure 2, though the container is up and running. After user re-deploy the container, user will get accurate results.
```

Reviewer: Refining Karel's suggestion
```suggestion
If you want to start the stopped container, you can get the results as shown in the following figure. However, the container is up and running. You can get accurate results after you re-deploy the container. 
```

> image::ReleaseNotes/stop_container.png[align="left", title="Stop container"]

Reviewer: ```suggestion

```

Author: I have to keep the pic, because of start container and stop container pic has different error message.

Reviewer: I suggested moving the stop container image after the first sentence, where the image is described. Please see the suggestion shared above.
